src/main/java/me/roinujnosde/titansbattle/BaseGame.java

```diff
@@ -67,6 +68,12 @@ public void start() {
         if (!getConfig().locationsSet()) {
             throw new IllegalStateException("You didn't set all locations!");
         }
+        ForkJoinPool.commonPool().execute(() -> {
```

Comment:
This should be moved inside Webhook class and replaced with Bukkit.getScheduler().runTaskAsynchronously()

Reply:
I really messed up, I'll fix it